typhon/arts/griddedfield.py

```diff
@@ -513,6 +513,7 @@ def to_xarray(self):
                 if len(coor)>0})
         if self.name is not None:
             da.name = self.name
+        da.attrs['data_name'] = self.dataname
```

Comment:
You could consider to name the attribute ``'dataname'`` for consistency, but that's up to you

Reply:
Thanks, I rather stick with underscores to stay consistent with the naming of attributes in cf conventions, for example.